src/main/java/no/digipost/signature/client/core/internal/ClientHelper.java

```diff
@@ -163,8 +164,10 @@ public XMLDirectSignatureJobStatusResponse getDirectStatusChange(Optional<Sender
 
     private <RESPONSE_CLASS> RESPONSE_CLASS getStatusChange(final Optional<Sender> sender, final Target target, final Class<RESPONSE_CLASS> responseClass) {
         return call(() -> {
-            Invocation.Builder request = httpClient.signatureServiceRoot().path(target.path(getActualSender(sender)))
+            Sender actualSender = getActualSender(sender, globalSender);
+            Invocation.Builder request = httpClient.signatureServiceRoot().path(target.path(actualSender))
                     .request()
+                    .header("polling_queue", actualSender.getPollingQueue().value)
```

Comment:
Bare legg til header hvis køen `!= null`.

Reply:
Det er forøvrig et vanlig pattern å navngi headers med bindestrek som skilletegn og store bokstaver i hvert ord (f. eks. `Content-Type` og `User-Agent`). Altså `Polling-Queue` :)

Reply:
Stryk første kommentar, headeren legges bare til hvis den har verdi 👍